clickhouse/columns/column.cpp

```diff
@@ -0,0 +1,38 @@
+#include "column.h"
+
+namespace clickhouse {
+
+bool Column::LoadPrefix([[maybe_unused]] InputStream* input, [[maybe_unused]] size_t rows) {
+    /// does nothing by default
+    return true;
+}
+
+bool Column::LoadBody([[maybe_unused]] InputStream* input, [[maybe_unused]] size_t rows) {
+    /// does nothing by default
+    return true;
+}
+
+bool Column::Load(InputStream* input, size_t rows) {
+    return LoadPrefix(input, rows) && LoadBody(input, rows);
+}
+
+void Column::SavePrefix([[maybe_unused]] OutputStream* output) {
+    /// does nothing by default
+}
+
+void Column::SaveBody([[maybe_unused]] OutputStream* output) {
+    /// does nothing by default
+}
+
+void Column::SaveSuffix([[maybe_unused]] OutputStream* output) {
+    /// does nothing by default
+}
+
+/// Saves column data to output stream.
+void Column::Save(OutputStream* output) {
+    SavePrefix(output);
```

Comment:
Please add a comment inside a function body explaining why we have to split saving in 3 steps. The best comment would also explain why this is done in CH, not only stating something like "follow CH serialization schema"

Reply:
Ok. I don't know all the reasons it was split in three parts, but I'll mention the Array(LowCardinality) example.